src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php

```diff
@@ -53,6 +53,7 @@ public function __construct(CacheClearerInterface $cacheClearer, Filesystem $fil
     protected function configure()
     {
         $this
+            ->setAliases(['cc'])
```

Comment:
Because the command is lazy-loaded, the right way to give it an alias is to add an additional `console.command` tag on its service definition.